src/Pipeline.cpp

```diff
@@ -673,70 +674,46 @@ struct JITFuncCallContext {
 
 }  // namespace
 
+struct Pipeline::JITCallArgs {
+    size_t size{0};
+    const void *fixed_store[64];
+    const void **store;
+
+    JITCallArgs(size_t size) : size(size) {
+        if (size > (sizeof(fixed_store) / sizeof(fixed_store[0]))) {
+            // TODO(zalman): Call new[]?
```

Comment:
why not just have a `std::vector<uint8_t>` member to use for storage?

Reply:
Because I am not convinced it is free on all platforms if I don't actually use it.